src/ui/Facet/FacetSettings.ts

```diff
@@ -164,7 +170,7 @@ export class FacetSettings extends FacetSort {
       $$(this.clearStateSection).toggle(!Utils.isNullOrUndefined(this.facetStateLocalStorage.load()));
     }
 
-    _.each(this.enabledSorts, (criteria: IFacetSortDescription, i) => {
+    each(this.enabledSorts, (criteria: IFacetSortDescription, i) => {
```

Comment:
When refactoring the underscore imports, should we take the opportunity to change to native `.forEach` `.map()` `.filter()` and so on?

Reply:
Native are a bit different than underscore version (eg: they handle undefined/null values a bit differently). They are also not all supported by IE11.

To change those, I'd need to be much more careful in checking that it's not breaking anything.